.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2
+jobs:
+  build:
+    parallelism: 4
```

Comment:
Does this help you?

Reply:
What do you mean? It increases the parallelism so should split the specs over 4 containers to finish much faster. This project has an advantage in that it can use open source containers so we're not quite as constrained as we are on our private projects (where I believe we usually use 2 as our parallelism).

Reply:
Ahhh I just didn’t see the circle test glob bit. :+1: